src/v3/utils.ts

```diff
@@ -110,6 +110,16 @@ export const createTokenId = (
   }
 }
 
+// Ignore requests with a future time with an allowance for clients that are close enough
+const isFutureTime = (
+  rateTime: number,
```

Comment:
Just make rateTime be a Date object so you don't have to do getTime() everywhere you pass in the param.

to make sure you pass in the params in the right order, make it an object you pass with
```
{
  date: Date
  rightNow: Date
}
```